src/cargo/core/resolver/resolve.rs

```diff
@@ -213,7 +213,34 @@ unable to verify that `{0}` is the same as when the lockfile was generated
         &self.metadata
     }
 
-    pub fn dependencies_listed(&self, from: &PackageId, to: &PackageId) -> &[Dependency] {
+    pub fn extern_crate_name(
```

Comment:
This should also help racer which does this mapping manually atm: https://github.com/racer-rust/racer/blob/a89145e9deb5efd2b9fb75826e556a6c6f9d044d/src/racer/project_model.rs#L155-L158

Reply:
cc @kngwyu

Reply:
:+1: